source/blog/2016-07-25-ember-2-7-and-2-8-beta-released.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Ember.js 2.7 and 2.8 Beta Released
+author: Dan Gebhardt
+tags: Releases
+---
+
+Ember.js 2.7, a minor version release of Ember with backwards compatible
+changes, is released today.
+
+Ember.js 2.8 beta, the branch of Ember that will be released as stable in
```

Comment:
Should we also mention that this will be the next LTS release (6 weeks after it is stable)?